Controller/MediaController.php

```diff
@@ -44,27 +45,28 @@ public function getMedia($id)
     /**
      * @throws NotFoundHttpException
      *
-     * @param string $id
-     * @param string $format
+     * @param Request $request
+     * @param string  $id
+     * @param string  $format
      *
      * @return Response
      */
-    public function downloadAction($id, $format = 'reference')
+    public function downloadAction(Request $request, $id, $format = 'reference')
```

Comment:
I meant by keeping the old signature, and check the first parameter's type, but indeed, I overlooked the fact that this is an action and some magic is happening here. So this isn't possible.